lib/foreman/renderer.rb

```diff
@@ -57,6 +59,11 @@ def foreman_url(action = "built")
               :token     => (@host.token.value unless @host.try(:token).nil?), :kind => action
     end
 
+    def foreman_server_fqdn
+      config = URI.parse(Setting[:unattended_url])
+      config.host || request.host
```

Comment:
Which seems to make sense for both new helpers. The only thing that makes it confusing is that our existing `foreman_url` helper uses `unattended_url` setting while the new one would use `foreman_url` setting. Anyway I've added two new helpers, please take a look.